src/Native/Unix/configure.cmake

```diff
@@ -709,6 +709,14 @@ check_c_source_compiles(
     "
     HAVE_IN_EXCL_UNLINK)
 
+check_c_source_compiles(
+    "
+    #include <netinet/tcp.h>
+    int main() { int x = TCP_KEEPALIVE; return x; }
+    "
+    HAVE_TCP_H_TCP_KEEPALIVE
+)
```

Comment:
Nit: Format it as above case `HAVE_IN_EXCL_UNLINK`.

Reply:
Will do, but as per @stephentoub [comment](https://github.com/dotnet/corefx/pull/29963/files/e2b75cc0c82619e61ee43418f9be580f9342aec1#r191559823) I copied the format from [this](https://github.com/luigiberrettini/corefx/blob/e2b75cc0c82619e61ee43418f9be580f9342aec1/src/Native/Unix/configure.cmake#L546)